ffwd-client/src/test/java/com/spotify/ffwd/v1/MetricTest.java

```diff
@@ -94,7 +94,8 @@ public void testSerializeDistributionValue() throws InvalidProtocolBufferExcepti
     final byte [] pointVal =  DISTRIBUTION_DATA_POINT.getBytes(StandardCharsets.UTF_8);
     ByteBuffer byteBuffer = ByteBuffer.allocate(pointVal.length);
     byteBuffer.put(pointVal);
-    Metric metricV1 = (new Metric()).value(Value.distributionValue(byteBuffer));
+    Metric metricV1 = (new Metric()).value(Value
+        .distributionValue(ByteString.copyFrom(byteBuffer.array())));
```

Comment:
Nit: ByteString.copyFrom(byteBuffer) might be cleaner, but perhaps you would then first need to flip the buffer

Reply:
Actually - just do ByteString.copyFromUtf8(DISTRIBUTION_DATA_POINT) instead